src/ebay-menu/__tests__/index.spec.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
-import { render, fireEvent } from '@testing-library/react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import { act } from '@testing-library/react-hooks'
```

Comment:
Act and screen are not being used do we need to import it?

Reply:
@HenriqueLimas thanks, removed. I've created an issue regarding that problem recently: #202